frontend/src/app/supportAdmin/DeviceType/ManageDevicesForm.test.tsx

```diff
@@ -73,7 +73,7 @@ describe("ManageDeviceTypeForm", () => {
   });
```

Comment:
could add name and test length to `"disables all input fields when no device is selected"`